lib/gds_api/email_alert_api.rb

```diff
@@ -116,12 +116,13 @@ def unsubscribe_subscriber(id)
   # Subscribe
   #
   # @return [Hash] subscription_id
-  def subscribe(subscriber_list_id:, address:, frequency: "immediately")
+  def subscribe(subscriber_list_id:, address:, frequency: "immediately", skip_confirmation_email: false)
```

Comment:
Just to be clear, this will send the confirmation email by default? (Presumably so the accounts stuff keeps on working)

Reply:
The email system should send an email by default, but this allows accounts to turn it off because they send their own confirmation email as part of their setup/subscribe process. Does that make sense?

Reply:
Ah, right - I had it backwards. Thanks for the clarification 👍